capella_model_explorer/components.py

```diff
@@ -302,6 +306,7 @@ def report_loader(
             template_id=template_id,
             model_element_uuid=model_element_uuid,
         ),
+        hx_headers='{"Vary": "' + vary_value + '"}',
```

Comment:
The Vary header accepts a list of header names, not any arbitrary values: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Vary

Reply:
Thank you!

I am going to consider:

1. capella-model-explorer version
2. capellambse version
3. context-diagrams version
4. jinja template file hash
5. model branch
6. model commit hash

Reply:
Is there a reason to consider the model branch? The model commit hash should be unique already (unless two braches are on the same state, but this should result in the same output)